### ISSUE TITLE
Context Menu Item for Find

### ISSUE THREAD

**Issue by Chips1234** [microsoft/terminal#5633]
# Description of the new feature/enhancement
A menu flyout item for the find feature.

**Comment by zadjii-msft** [comment]
Thanks for the offer of help! I think we're trying pretty hard ad the moment to _not_ add additional UI elements to the New Tab flyout we already have. It's really the only UI we have at the moment, but it's a slippery slope towards having just every button in that single UI, and all of a sudden it's a cluttered mess like this:
![image](https://user-images.githubusercontent.com/18356694/80601259-3061de80-89f3-11ea-9d38-b11ebbb428e7.png)

Hopefully once #3789 lands (in 1.1), then we'll be able to add menu entries to the right-click menu on a tab, and I think that's probably a better place to put something like this.

Then there's also #3337 for adding a context menu to the terminal controls themselves, which would also be an appropriate place for this

Then of course, there's always #5400, which is another possible place.


I'm going to leave this open as a potential follow-up task of either #3337 or #3789

**Comment by Chips1234** [comment]
Alright, we'll talk about it when context menu is introduced.

**Comment by Chips1234** [comment]
> Thanks for the offer of help! I think we're trying pretty hard ad the moment to _not_ add additional UI elements to the New Tab flyout we already have. It's really the only UI we have at the moment, but it's a slippery slope towards having just every button in that single UI, and all of a sudden it's a cluttered mess like this:
> ![image](https://user-images.githubusercontent.com/18356694/80601259-3061de80-89f3-11ea-9d38-b11ebbb428e7.png)

Is that Microsoft saying that their own products are a mess? Lol

**Comment by DHowett-MSFT** [comment]
It would not be wise to take the opinion of one person at a company with multiple tens of thousands of employees as representative of the entire company.

**Comment by Chips1234** [comment]
I understand. It was an innocent joke.

**Comment by Chips1234** [comment]
Alright. It's now moved to #5835.

**Comment by DavidJRWang** [comment]
In regards to the above discussion, has it been determined whether or not this is something that is wanted?

**Comment by zadjii-msft** [comment]
Honestly, I don't remember. I thought I closed this because _eventually_ we were going to make it possible for users to add this on their own. I don't remember why I tagged this easy starter...



...


Okay, now I'm remembering. We thought it was a fine idea for someone to add to the tab context menu
![image](https://user-images.githubusercontent.com/18356694/141859582-0c10390e-32f0-4b3d-af2b-d3e7705a7932.png)

And that should be relatively easy to do.
